src/rdf4cpp/rdf/util/TriBool.hpp

```diff
@@ -0,0 +1,96 @@
+#ifndef RDF4CPP_UTIL_TRIBOOL_HPP
+#define RDF4CPP_UTIL_TRIBOOL_HPP
+
+#include <array>
+
+namespace rdf4cpp::rdf::util {
+
+struct TriBool {
```

Comment:
Implementation is good. Only some docstrings for the class, public members and public methods are missing.

Reply:
added comments to the conversions and partial_ordering_eq (also moved that out of the struct; as it didn't really belong there)

Reply:
Nothing beautiful but some candidates to pick from:
- `pw_ordering_eq`
- `partial_weak_ordering_eq`

BTW: it should also be marked as `noexcept`. Implicit `noexcept` from `constexpr` might be lost when it is called at runtime AFAIK.

Reply:
same for the two constructors I think